src/routes/meetups.ts

```diff
@@ -18,6 +18,24 @@ export const meetups: TMeetup[] = [
 	// 		},
 	// 	],
 	// },
+	{
+		dateISO: '2024-12-16T18:00:00.000Z',
+		time: '18:00 - 21:00',
+		name: '2024/12 Svienna - Svelte Society Vienna',
+		eventLink: 'https://www.meetup.com/svelte-society-austria/events/304285450/',
+		pictureCount: 0,
+		talks: [
+			{
+				name: 'Super secret talk',
```

Comment:
Actually we know that @Lms24 will do `Astro for Svelte Developers`

Reply:
@jycouet added the talk, thx :)